homework-1.html

```diff
@@ -0,0 +1,130 @@
+    Home Improvement <input type="checkbox" class="tags" name="Home Improvement"><br>
+    Food and Household <input type="checkbox" class="tags" name="Food and Household"><br>
+    Luxury <input type="checkbox" class="tags" name="Luxury"><br>
+    Electronics <input type="checkbox" class="tags" name="Electronics"><BR>
+    Gardening <input type="checkbox" class="tags" name="Gardening"><br>
+    Fashion <input type="checkbox" class="tags" name="Fashion"><br>
+    </p>
+    <p> <h3>Stores:</h3>
+      Walmart <input type="checkbox" class="stores" name="Walmart"><br>
+      Target <input type="checkbox" class="stores" name="Target"><br>
+      Whole Foods <input type="checkbox" class="stores" name="Whole Foods"><br>
+      Home Depot <input type="checkbox" class="stores" name="Home Depot"><br>
+      Amazon <input type="checkbox" class="stores" name="Amazon"><br>
+      Costco <input type="checkbox" class="stores" name="Costco"><br>
+      Walgreens <input type="checkbox" class="stores" name="Walgreens"><br>
+    </p>
+    <button id="add">Sumbit Item</button>
+    <h2>Or look at the inventories of some stores</h2>
+      Walmart <input type="checkbox" class="storesToLookAt" name="Walmart"><br>
+      Target <input type="checkbox" class="storesToLookAt" name="Target"><br>
+      Whole Foods <input type="checkbox" class="storesToLookAt" name="Whole Foods"><br>
+      Home Depot <input type="checkbox" class="storesToLookAt" name="Home Depot"><br>
+      Amazon <input type="checkbox" class="storesToLookAt" name="Amazon"><br>
+      Costco <input type="checkbox" class="storesToLookAt" name="Costco"><br>
+      Walgreens <input type="checkbox" class="storesToLookAt" name="Walgreens"><br>
+    <button id="look">Look at  these stores</button>
+    <p id="storeInventories"></p>
+    <script>
+    class Store {
+      constructor(name) {
+        this.name=name;
+        this.items=[];
+      }
+      additem(item){
+        this.items.push(item);
+      }
+    }
+    const listofstores=[new Store("Walmart"),new Store("Target"),new Store("Whole Foods"),new Store("Home Depot"),new Store("Amazon"),new Store("Costco"),new Store("Walgreens")]
+    let listofitems=[]
+    class Item {
+      constructor(name,price,tags,stores) {
+        this.name = name;
+        this.price = price;
+        this.tags = tags
+        this.stores = stores;
+
+        for(let i=0;i<stores.length;i++){
+          for(let j=0;j<listofstores.length;j++){
+            if(listofstores[j].name==stores[i]){
+              listofstores[j].items.push(name);
+            }}}}}
+            $.ajax({
+              url: "https://data.sparkfun.com/output/5J4bbngozmtlgx0mrJ0b.json",
+              success: function(result){
+                for(let i=0;i<result.length;i++){
+                  result[i].stores = result[i].stores.split(",");
+                  listofitems.push(new Item(result[i].name,result[i].price,result[i].tags,result[i].stores));
```

Comment:
```js
listofitems.push(new Item(result[i].name, result[i].price,result[i].tags, result[i].stores));
```

Let things breathe. Space after comma. Also the variable name is not following convention. It's all lowercase. JavaScript follows camelCase.